helm/chart/maesh/charts/metrics/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v1
 description: A metrics Helm chart for Kubernetes
 name: metrics
-version: 0.0.6
+version: 0.1.0
```

See my previous comment.

```suggestion
version: 0.0.6
```